concepts/packages/introduction.md

```diff
@@ -1,52 +1,30 @@
 # Introduction
 
-TODO: the content below is copied from the exercise introduction and probably needs rewriting to a proper concept introduction
-
-## packages
-
-## functions
-
-## variables
-
-In Go an application is organized in packages. A package is a collection of source files located in the same folder. All source files in a folder must have the same package name at the top of the file. The package name is preferred to be the same as the folder it is located in.
+In Go an application is organized in packages.
+A package is a collection of source files located in the same folder.
+All source files in a folder must have the same package name at the top of the file.
+The package is preferred to be the same as the folder it is located in.
 
 ```go
 package greeting
 ```
 
-Go is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable can be defined either by explicitly specifying its type, or by assigning a value to have the Go compiler infer its type based on the assigned value.
-
-```go
-var explicit int // Explicitly typed
-implicit := 10   // Implicitly typed
-```
-
-The value of a variable can be assigned using the `:=` and updated using the `=`. Once defined, a variable's type can never change.
+Go provides an extensive standard library of packages which you can use in your program using the `import` keyword.
+Standard library packages are imported using their name.
 
 ```go
-count := 1 // Assign initial value
-count = 2  // Update to new value
+package greeting
 
-// Compiler error when assigning different type
-// count = false
+import "fmt"
 ```
 
-A function can have zero or more parameters. All parameters must be explicitly typed, there is no type inference for parameters. A function can also have multiple return values which must also be explicitly typed. Values are returned from functions using the `return` keyword. To allow a function to be called by code in other packages, the name of the function must start with a capital letter.
+Multiple packages can be imported at once by encoding them in brackets.
```

Comment:
The importing of multiple packages is not needed to solve the exemplar solution. Quoting [the docs](https://exercism.org/docs/building/tracks/concept-exercises):

> Only information that is needed to understand the fundamentals of the concept and solve the exercise should be provided. Extra information should be left for the concept's about.md document.

This content is perfect for the `about.md` file, but should be removed from this `introduction.md` file.